armi/operators/tests/test_operators.py

```diff
@@ -103,37 +111,96 @@ def test_loadStateError(self):
         with self.assertRaises(RuntimeError):
             self.o.loadState(0, 1)
 
+    def test_setStateToDefault(self):
+
+        # reset the runType for testing
+        self.assertEqual(self.o.cs[CONF_RUN_TYPE], "Standard")
+        self.o.cs = self.o.cs.modified(newSettings={"runType": "fake"})
+        self.assertEqual(self.o.cs[CONF_RUN_TYPE], "fake")
+
+        # validate the method works
+        cs = self.o.setStateToDefault(self.o.cs)
+        self.assertEqual(cs[CONF_RUN_TYPE], "Standard")
+
+    def test_snapshotRequest(self):
+        with TemporaryDirectoryChanger():
+            self.o.snapshotRequest(0, 1)
```

Comment:
```suggestion
            self.o.snapshotRequest(0, 1)
            self.assertTrue(os.path.exists("snapShot0_1"))
```

I abandoned a PR where I added an assert statement to this test. It's unrelated to your PR, but wanna sneak it in for me?